gslab_scons/record_dir.py

```diff
@@ -0,0 +1,113 @@
+import os
+import scandir
+import mmh3
+
+from misc import check_and_expand_path, make_heading
+
+def record_dir(start_path, name, 
+               include_checksum = False,
+               file_limit = 5000, 
+               logpath = 'state_of_external.log'):
```

Comment:
`logpath` vs `start_path` kinda throws me off but also it's the kind of syntax I would personally use so who knows, you decide.

Reply:
Now `outpath` and `inpath` respectively. Hard to mix those up.